LondonBritishLibrary/orient/BLorient692.xml

```diff
@@ -177,143 +177,288 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                                            <persName ref="PRS5616IyasuI"><sic>አድም፡</sic> ሰገደ፡ ኢያሱ፡</persName> ብሔረ፡ 
+                                            ንብረቶሙ፡ በጽንፈ፡ <placeName ref="LOC5148Qaha">ቀሐ፡</placeName> ወእምድኅረ፡ ብዙኅ፡ ዘመን፡ ዓርገ፡
+                                            <persName ref="PRS5616IyasuI"><sic>አድም፡</sic> ሰገደ፡ ኢያሱ፡</persName> ብሔረ፡
                                             <placeName ref="LOC5597Sawa">ሽዋ፡</placeName> ወቀተሎ፡ ለአዳ፡ ወተዓየነ፡ ላዕለ፡ ደብረ፡ ብርሃን፡
                                     ወረከብዎ፡ ኃያላነ፡ ንጉሥ፡ ማእከለ፡ ቀትል፡ ለነጋሢ፡ ኃያል፡ ወጽኑዕ፡ ዘአዳም፡ ራዕዩ፡ ወነዊኅ፡ ቆሙ፡ ወሶበ፡ ርእዮ፡ ንጉሥ፡ አፍቀሮ፡ ጥቀ፡ ወአምጽኦ፡
                                     ብሔረ፡ ጐንደር፡ ወጐንደዮ፡ ሕቀ፡ ሤሞ፡ ውስተ፡ ሀገሩ፡ ወገቢኦ፡ ተጻብአ፡ ምስለ፡ የዲ፡ ወቀተሎ፡ ነጋሢ፡ ወአመ፡ መንግሥቱ፡ ለ<persName ref="PRS2377Bakaffa">በካፋ፡</persName> መጽአ፡ ኀበ፡ ንጉሥ፡
-                                            ውስተ፡ ጐንደር፡ <persName ref="PRS8666Sebestya" cert="low">ስብስትያኖስ፡</persName> ወኃደረ፡ ላዕለ፡ ሀገሩ፡ አቦራ፡ ወተቀበልዎ፡ አዝማዲሁ፡ 
-                                            በፍሥሓ፡ 
-                                            ወበሐሤት፡ ወርእያ፡ ለማርያም፡ ወለተ፡ ወለታ፡ 
+                                            ውስተ፡ ጐንደር፡ <persName ref="PRS8666Sebestya" cert="low">ስብስትያኖስ፡</persName> ወኃደረ፡ ላዕለ፡ ሀገሩ፡ አቦራ፡ ወተቀበልዎ፡ አዝማዲሁ፡
+                                            በፍሥሓ፡
+                                            ወበሐሤት፡ ወርእያ፡ ለማርያም፡ ወለተ፡ ወለታ፡
                                     አፍቀራ፡ ጥቀ፡ ወአንበራ፡ ምስሌሁ፡ ወእንዘ፡ ይሠየም፡ መርዕድ፡ አዝማች፡ ተመይጠ፡ ውስተ፡ ሀገሩ፡ ወአምጽኣ፡ ምስሌሁ፡ ወወለደት፡ ሎቱ፡ ደቂቀ፡
-                                    ወአዋልደ፡ እስከ፡ ዝየ፡ ይበውኡ፡ ወይወጽኡ፡ መሳፍንተ፡ ሸዋ፡ <persName ref="PRS1878AmmehaI">አምኃ፡ <sic>ኢየሱስስ፡</sic></persName> እስመ፡ 
+                                    ወአዋልደ፡ እስከ፡ ዝየ፡ ይበውኡ፡ ወይወጽኡ፡ መሳፍንተ፡ ሸዋ፡ <persName ref="PRS1878AmmehaI">አምኃ፡ <sic>ኢየሱስስ፡</sic></persName> እስመ፡
                                             ኢመጽአ፡ ተረስዓ፡ ዝክረ፡ ልደት፡ <foreign xml:lang="am">ይኽነን፡ ታሪክ፡ የጻፋ፡ ከዚህ፡
                                                 መጽሐፍ፡ አዛዥ፡ <persName ref="PRS13795Mazmure">መዝሙሬ፡</persName> ናቸው፡ ጸሐፌ፡ ትእዛዝ፡ ዘንጉሥ።</foreign>
                                     </q>
                                 </item>
                             </list>
-                            
+
                         </additions>
-                        
+                        <bindingDesc>
+<binding contemporary="true" xml:id="binding">
+<decoNote xml:id="b1">Two wooden boards, covered by recent (from the library) brown leather (tooled on the spine and on part of the boards). On the leather spine is inscribed the manuscript`s title (in block letters): 'ACTS OF MARTYRS, AND HOMILIES. AETHIOPIC. BRIT. MUS. ORIENTAL 692.'</decoNote>
+<decoNote xml:id="b2" type="bindingMaterial">
+<material key="wood"></material>
+<material key="leather"></material>
+</decoNote>
+<decoNote xml:id="b3">More than two pairs of sewing stations (probably 5, difficult to understand).</decoNote>
```

Comment:
= 5 sewing stations or 5 pairs of sewing stations? This sentence is ambiguous. I suggest: "Probably 5 sewing stations (difficult to discern)" or "Probably 5 sewing stations (difficult to discern from the images)".

Reply:
It was difficult to understand also  in situ.